src/scenic/domains/driving/roads.py

```diff
@@ -1335,6 +1348,29 @@ def show(self, labelIncomingLanes=False):
                     units="dots",
                     color="#A0A0A0",
                 )
+
+            for lg in road.laneGroups:
+                curb = lg.curb
+                if curb.length >= 40:
+                    cpts = curb.pointsSeparatedBy(20)
+                else:
+                    cpts = [curb.pointAlongBy(0.5, normalized=True)]
+                chs = [curb.orientation[pt].yaw for pt in cpts]
+                cx, cy, _ = zip(*cpts)
+                cu = [math.cos(h + (math.pi / 2)) for h in chs]
+                cv = [math.sin(h + (math.pi / 2)) for h in chs]
+                plt.quiver(
+                    cx,
+                    cy,
+                    cu,
+                    cv,
+                    pivot="middle",
+                    headlength=4.5,
+                    scale=0.06,
+                    units="dots",
+                    color="#FF4444",
```

Comment:
Let's use the same color as for the curb outline (`#606060` I think) so that the arrows aren't confused with the red lane boundaries. Actually I'm not sure if we should show these arrows by default at all: it might be better to not show them to avoid confusion with the lane centerlines (which convey the direction information anyway). You could add a keyword argument to `show` to turn them on so that we can do that for debugging purposes in the future, but have them off by default.